experiments/conductor/cmd/runner/controller_commands.go

```diff
@@ -387,6 +387,73 @@ Respond only with the YAML content, no explanations.`,
 	}, nil, nil
 }
 
+func moveTestToSubDir(ctx context.Context, opts *RunnerOptions, branch Branch, execResults *ExecResults) ([]string, *ExecResults, error) {
+	log.Printf("Moving test to sub directory for %s", branch.Name)
+
+	// Check if we have the required fields
+	if branch.Group == "" {
+		return nil, nil, fmt.Errorf("branch %s is missing Group field", branch.Name)
+	}
+
+	if branch.Kind == "" {
+		return nil, nil, fmt.Errorf("branch %s is missing Kind field", branch.Name)
+	}
+
```

Comment:
Should probably have a check which skips if the destination directory already exists.

Reply:
Done!